runchecks.sh

```diff
@@ -16,15 +20,10 @@ echo "Running isort"
 isort --check-only --diff --quiet \
  || { echo "isort errors found! Run 'isort' with no options to fix."; exit 1; }
```

Comment:
For some reason `isort` is no longer using the `setup.cfg` configuration

Reply:
How can you notice that it is not?

Reply:
when I run `./runchecks.sh` in the docker image, a number of isort errors are shown for directories that should be ignored

Reply:
I have fixed this in https://github.com/mozilla/treeherder/pull/6134